### ISSUE TITLE
Техническая поддержка (модерирование)

### ISSUE THREAD

**Issue by mj4444ru** [freedomsex/freedomsex#25]
Необходимо добавить модерирование работы технической поддержки.
Может сейчас ещё никто не гадил, то в будущем это вполне возможно и это будет крайне печально для сайта в целом, ибо ответы ТП считаются ответами хозяев этого сайта и это лицо этого сайта.

Необходимо добавить модерирование поддержки и возможность временно запретить пользователю отвечать в службе поддержки в случае если появится вредитель.
Если вы видите в этом смысл, могу подумать над более подробной реализацией.

**Comment by tebaly** [comment]
Да. Мы сейчас задумались над разделением "административной части" и части отвечающей за "помощь". Содействуйте конечно, ни от какой помощи мы никогда не отказываемся. Проект начинает публичное развитие и совместную разработку. Вы вправе не ждать одобрения.

**Comment by inquisitor6789** [comment]
было такое что и гадили.
основная проблема тех.пода в том, что сообщения от Юли обезличены и понять кто отвечал может только разработчики. Если оставить идентификатор отвечающего, то полномочий админа хватит чтобы его рейтинг увести в минус

**Comment by mj4444ru** [comment]
Реализовано же

**Comment by inquisitor6789** [comment]
только тут http://qbsex.ru/history/rehelp/

**Comment by mj4444ru** [comment]
@inquisitor6789, будешь за старшего... у тя эта ссылка есть и больше никому не надо )))

**Comment by Fenslau** [comment]
На самом деле, пока и правда эту ссылку стоит в секрете подержать с недельку...Посмотреть, кто как и на что отвечает. Остальные не хотят принимать участие в обсуждении этом. А кто захочет - увидит ее.

**Comment by tebaly** [comment]
Секрета особого нет, просто beta

**Comment by inquisitor6789** [comment]
@mj4444ru, вообще-то это по твоей просьбе сделали https://github.com/tebaly/freedomsex/issues/142
@tebaly, ай-ай-ай, не хорошо обманывать :-)
> Кроме как здесь, ссылки на этот инструмент нигде нет, и кроме вас о нем никто не знает

**Comment by tebaly** [comment]
Не придирайтесь к словам, забыл поставить тэг `<irony>` :) 
* фитча тестируется, сейчас выкатим на все сервера, и добавим вкладку в разделе помощи. Просто на время `мусора в логе` и тд, хотелось не афишировать - инструмент крайне сырой.

**Comment by mj4444ru** [comment]
Так решили проблему частично #142
Конечно не так как хотелось бы, но всё же...

**Comment by tebaly** [comment]
Если мне не изменяет память @13044460 резидент по вашей рекомендации(22290748) - от него исходит "Мы 10 пацанов. Хочешь доставить нам удовольствие? Сымай штаны и подставляй жопу."

Про "субсидиарную ответственность" я еще не писал, но администратор выдвинувший кандидата в резиденты несет за его действия _определенную_ ответственность.
* Вот что с вами делать?

**Comment by tebaly** [comment]
То что обратная связь налажена через ту самую жопу не повод оскорблять Пользователей. Они не всегда в курсе, что _Юлия_ робот. На главной есть ссылка с описанием всей "ситуации"
* **Помощник по сайту, спросить** - и есть справка, где описано, что это частично робот, так как отвечает не девушка, а группа поддержки. И писать она могла сама, пока не отключили.

Нет никакой надобности писать "Мы 10 пацанов админов..." - это троллинг и саботаж СБ, так нельзя.
* Игнорируйте бессмысленные обращения, закрывайте без ответа.

---
1304446 - учетка сугубо для доступа в админку. Резидент по рекомендации 22290748. На данный момент доступ к администрированию ограничен, статус резидента снят. Если кто-то поддерживает связь, свяжитесь пожалуйста, чтобы прояснить ситуацию. Предложите пожалуйста свои варианты решения конфликта.

**Comment by tebaly** [comment]
#373 - пользователи могут писать Юле самостоятельно, изначально не зная что это помощник. Их вины в этом нет. Для этих случаев есть текст **"Пожалуйста, не знакомьтесь со мной)) Я же виртуальный-помощник. И я не могу с вами познакомиться.."** 

Юлию иногда "используют" для тестирования функций сайта.

**Comment by tebaly** [comment]
@inquisitor6789 @Fenslau @DrAllcome - вам что-нибудь известно по теме?

**Comment by inquisitor6789** [comment]
нет

**Comment by mj4444ru** [comment]
Прошу прощения. Знал о проблеме, но видимо недостаточно уделил внимания. Ввиду моего отсутствия не был в курсе.
